### PR TITLE
fix: unwanted padding on pfe-accordion

### PR DESCRIPTION
**Pull request by deborapozzebon**
### Related issue

- (#883) Pfe-accordion: disclosure includes unwanted padding in the panel

### Preview

Link(s) to demo page(s) where this element can be viewed:
- [Preview link](https://5e6089f7c8e38b0008963801--happy-galileo-ea79c4.netlify.app/elements/pfe-accordion/demo/) 

### What has changed and why

- Updating padding to match the design specifications

### Testing instructions

In the demo page, inspect the disclosure element, check that the padding is diffenrent between the left and rigth side. Check that the right padding is greater than twice the left padding value. After the changes, the right and left padding values are the same.

#### Browser requirements

Your component should work in all of the following environments:

- [x] Latest 2 versions of Edge
- [ ] Internet Explorer 11 (should be useable, not pixel perfect)
- [ ] Latest 2 versions of Firefox (one on Mac OS, one of Windows OS)
- [ ] Firefox 68 (or latest version for Red Hat Enterprise Linux distribution)
- [ ] Latest 2 versions of Chrome (one on Mac OS, one of Windows OS)
- [ ] Latest 2 versions of Safari
- [ ] Android mobile device (such as the Galaxy S9)
- [ ] Apple mobile device (such as the iPhone X)
- [ ] Apple tablet device (such as the iPhone Pro)

### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.

- [x] Expected files: all files in this pull request are related to one request or issue (no stragglers or scope-creep).
- [ ] Tests have been updated to cover these changes.
- [ ] Browser testing passed.
- [ ] Repository compiles and tests pass.
- [ ] Changelog updated (not needed for documentation updates).
- [ ] Documentation (README.md, WHY.md, etc.) updated or added.
- [ ] Link to the demo recording: []()
- [ ] Approved by designer.

### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

**Comment by deborapozzebon** [comment]
Hello @castastrophe. This is the fix that we discussed on Slack. Could you please take a look? Thanks

File: elements/pfe-accordion/src/pfe-accordion-panel.scss [hunk 1]
```diff
@@ -27,7 +27,7 @@
   :host([pfe-disclosure="true"]) & {
     @include pfe-print-local((
       panel-container: (
-        Padding: 0 calc(#{pfe-local(Padding, $region: base)} * 3) pfe-local(Padding, $region: base) calc((#{pfe-local(Padding, $region: base)} * 1.5) - 4px)
+        Padding: 0 calc(#{pfe-local(Padding, $region: base)} * 1.5) pfe-local(Padding, $region: base) calc((#{pfe-local(Padding, $region: base)} * 1.5))
```
Comment (castastrophe) [inline elements/pfe-accordion/src/pfe-accordion-panel.scss:30]:
```suggestion
        Padding: 0 calc(#{pfe-local(Padding, $region: base)} * 1.5) pfe-local(Padding, $region: base)
```

@deborapozzebon This is looking good!  You can use the shorthand here to make it a little less verbose if you like.  Will result in the same output.

**Comment by castastrophe** [comment]
This update looks great, thanks for tackling it.  If you wouldn't mind updating the `CHANGELOG-prerelease.md` with your update.  And write out a test case in the PR description.  Something like: open the demo page, inspect the disclosure element, check that the padding is...etc.  Let me know if you have any questions or get stuck.  Also let me know if you need help validating the update in the listed browsers.  I have access to BrowserStack and can do that for you if you need me to.

**Review by kylebuch8 — APPROVED** [review]
Lopsided Game: Timeout Mandatory

**Comment by kylebuch8** [comment]
@deborapozzebon Congrats on the contribution and thank you for the help.

**Pull request merged**